### PR TITLE
Metrics update

### PR DESCRIPTION
**Pull request by theletterf**

### CODE REVIEW

File: metric-metadata/container.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+type: k8s/container
```
Comment (rmfitzpatrick) [inline metric-metadata/container.yaml:1]: I think the filenames should contain the receiver name as a prefix or parent dir for multiple metadata.yaml components like this or they are ambiguous

Comment (theletterf) [reply metric-metadata/container.yaml:1]: @pmcollins Is this something we can change in the script?

Comment (pmcollins) [reply metric-metadata/container.yaml:1]: I'm happy to continue helping with this, but would welcome any rotel involvement as well :-)

Comment (rmfitzpatrick) [reply metric-metadata/container.yaml:1]: https://github.com/splunk/collector-config-tools/issues/20

**Review by rmfitzpatrick — APPROVED** [review]

**Pull request merged**